src/AppEntry.js

```diff
@@ -1,31 +1,25 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { init, RegistryContext } from './store';
 import App from './App';
 import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers';
 import logger from 'redux-logger';
 
-const AppEntry = ({ hasLogger }) => {
-  const registry = hasLogger ? init(logger) : init(logger);
-  return (
-    <RegistryContext.Provider
-      value={{
-        getRegistry: () => registry,
-      }}
-    >
-      <Provider store={registry.getStore()}>
-        <Router basename={getBaseName(window.location.pathname, 1)}>
-          <App />
-        </Router>
-      </Provider>
-    </RegistryContext.Provider>
-  );
-};
+const registry = init(logger);
```

Comment:
It's not technically incorrect I would rather see the `init` called from `React.useMemo` with empty dependency array. Also the `logger` middleware is used on all environments (probably a bug from previous debugging), could be nice to use something like https://webpack.js.org/plugins/define-plugin/ here.

Reply:
Fixed. Because of errors importing from webpack, I'm using a runtime check of `process.env.NODE_ENV`. If